src/mavis/convert/vcf.py

```diff
@@ -122,6 +123,46 @@ def parse_bnd_alt(alt: str) -> Tuple[str, int, str, str, str, str]:
         raise NotImplementedError('alt specification in unexpected format', alt)
 
 
+def handle_edge_case(std_row: Dict) -> Dict:
```

Comment:
This is not a very descriptive function name. Rather than call this function twice below I would rename it to something like `convert_imprecise_break2` and put the if/else logic for CIEND vs CILEN in here as well. Then you can also add the explanation to the function description